### DISCUSSION TITLE
https?

### DISCUSSION THREAD

**Discussion by Nuube** [filebrowser/filebrowser#1912] [Q&A]
I just installed Filebrowser with docker but when I try to access the page via https:// I only get an error:

`ERR_SSL_PROTOCOL_ERROR`

Unsecure connections with http:// works like a charm.

Anything I am missing?

**Comment by ramiresviana** [comment]
HTTPS is not enabled by default, you can enable it by using `-key` and `-cert` parameters, you have to provide the path to the corresponding files, more information at https://filebrowser.org/cli/filebrowser. On Docker you can set these options by using environment variables or appending arguments to the entrypoint.

**Reply by nmincone** [reply]
I placed FileBrowser behind NPM with a wildcard cert, works perfectly.

**Comment by soulfulkhani** [comment]
> HTTPS is not enabled by default, you can enable it by using `-key` and `-cert` parameters, you have to provide the path to the corresponding files, more information at https://filebrowser.org/cli/filebrowser. On Docker you can set these options by using environment variables or appending arguments to the entrypoint.
how to do this in windows using windows terminal
